multi-node/generic/controller-install.sh

```diff
@@ -520,6 +520,296 @@ EOF
 EOF
     }
 
+    # For one controller and one worker node, fix memory at
+    # 224Mi. Multiple node clusters should scale up their memory by
+    # about 12Mi per node
+    local TEMPLATE=/srv/kubernetes/manifests/heapster-rc.json
+    [ -f $TEMPLATE ] || {
+        echo "TEMPLATE: $TEMPLATE"
+        mkdir -p $(dirname $TEMPLATE)
+        cat << EOF > $TEMPLATE
+{
+  "apiVersion": "v1",
+  "kind": "ReplicationController",
+  "metadata": {
+    "name": "heapster-v10",
+    "namespace": "kube-system",
+    "labels": {
+      "k8s-app": "heapster",
+      "version": "v10",
+      "kubernetes.io/cluster-service": "true"
+    }
+  },
+  "spec": {
+    "replicas": 1,
+    "selector": {
+      "k8s-app": "heapster",
+      "version": "v10"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "k8s-app": "heapster",
+          "version": "v10",
+          "kubernetes.io/cluster-service": "true"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "image": "gcr.io/google_containers/heapster:v0.18.2",
+            "name": "heapster",
+            "resources": {
+              "limits": {
+                "cpu": "100m",
+                "memory": "224Mi"
+              },
+              "requests": {
+                "cpu": "100m",
+                "memory": "224Mi"
+              }
+            },
+            "command": [
+              "/heapster",
+              "--source=kubernetes:''",
+              "--sink=influxdb:http://monitoring-influxdb.kube-system.svc.cluster.local:8086",
+              "--stats_resolution=30s",
+              "--sink_frequency=1m"
+            ]
+          }
+        ]
+      }
+    }
+  }
+}
+EOF
+    }
+
+    # This configuration leaves Grafana open without authentication (other than
+    # authentication via the Kubernetes API). In production, grafana
+    # authentication should be enabled.
+    local TEMPLATE=/srv/kubernetes/manifests/influxdb-grafana-rc.json
```

Comment:
Do we necessarily *need* grafana as a default addon? I agree that heapster is necessary because a core feature (horizontal scaling) requires it. However, I lean toward not installing optional applications, and leaving this decision up to user (and something we could possible add to documentation). Thoughts?